src/api/config/encoder.rs

```diff
@@ -15,6 +15,9 @@ use crate::api::{Rational, SpeedSettings};
 use crate::encoder::Tune;
 use crate::serialize::{Deserialize, Serialize};
 
+#[cfg(feature = "unstable")]
```

Comment:
I think you can avoid adding the `unstable` while refining the patch.

Reply:
Removed the `unstable` conditions.